src/server/actions/member-actions.ts

```diff
@@ -23,6 +30,10 @@ export const upsertMemberAction = async (member: Member | NewMember) => {
     return results[0]
```

Comment:
do we want to `executeTakeFirstOrThrow` for this call on line 15 instead of returning[0] and throwing an error manually? Might be simpler

Reply:
great point, thanks!